projects/scream/scream_post/translate_prognostic_output.py

```diff
@@ -0,0 +1,297 @@
+    "machine_learning_U_tend": "dQu",
+    "machine_learning_V_tend": "dQv",
+    "machine_learning_T_mid_tend": "dQ1",
+    "machine_learning_qv_tend": "dQ2",
+    "VapWaterPath": "water_vapor_path",
+    "ps": "PRESsfc",
+    "pseudo_density": "pressure_thickness_of_atmospheric_layer",
+    "surf_evap": "surf_evap",
+    "surf_sens_flux": "SHTFLsfc",
+    "surf_radiative_T": "TMPsfc",
+    "z_mid_at_500hPa": "h500",
+    "T_mid_at_850hPa": "TMP850",
+    "T_mid_at_700hPa": "TMP700",
+    "T_mid_at_500hPa": "TMP500",
+    "T_mid_at_200hPa": "TMP200",
+    "omega_at_500hPa": "w500",
+    "RelativeHumidity_at_850hPa": "RH850",
+    "RelativeHumidity_at_700hPa": "RH700",
+    "RelativeHumidity_at_500hPa": "RH500",
+    "SeaLevelPressure": "PRMSL",
+    "SW_flux_dn_at_model_bot": "DSWRFsfc",
+    "LW_flux_dn_at_model_bot": "DLWRFsfc",
+    "LW_flux_up_at_model_bot": "ULWRFsfc",
+    "SW_flux_up_at_model_bot": "USWRFsfc",
+    "SW_flux_up_at_model_top": "USWRFtoa",
+    "LW_flux_up_at_model_top": "ULWRFtoa",
+    "SW_flux_dn_at_model_top": "DSWRFtoa",
+}
+
+RENAME_ML_CORRECTION_TO_MACHINE_LEARNING = {
+    "mlcorrection_T_mid_tend": "machine_learning_T_mid_tend",
+    "mlcorrection_qv_tend": "machine_learning_qv_tend",
+    "mlcorrection_horiz_winds_tend": "machine_learning_horiz_winds_tend",
+}
+
+TEMP = "T_mid"
+SPHUM = "qv"
+DELP = "pseudo_density"
+EASTWARD_WIND = "U"
+NORTHWARD_WIND = "V"
+
+
+def _get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "input_data", type=str, default=None, help=("Input netcdf."),
+    )
+    parser.add_argument(
+        "output_path",
+        type=str,
+        help="Local or remote path where output will be written.",
+    )
+    parser.add_argument(
+        "process_label", type=str, help="Either nudging or machine_learning",
+    )
+    parser.add_argument(
+        "nudging_variables",
+        type=str,
+        help="List of nudging variables deliminated with commas",
+    )
+    parser.add_argument("chunk_size", type=int, help="Chunk size for output zarrs.")
+    parser.add_argument("save_to_disk", type=str, help="Either 2d or 3d.")
+    parser.add_argument(
+        "--subset",
+        help="Whether to subset to a smaller set of timesteps",
+        action="store_true",
+    )
+    return parser
+
+
+def compute_PRATEsfc(ds):
+    """
+    Compute PRATEsfc from precip_liq_surf_mass_flux and precip_ice_surf_mass_flux
+    """
+    if "PrecipIceSurfMassFlux" in ds.variables:
+        ds = ds.rename({"PrecipIceSurfMassFlux": "precip_liq_surf_mass_flux"})
+    if "PrecipLiqSurfMassFlux" in ds.variables:
+        ds = ds.rename({"PrecipLiqSurfMassFlux": "precip_ice_surf_mass_flux"})
+    PRATEsfc = (ds.precip_liq_surf_mass_flux + ds.precip_ice_surf_mass_flux) * 1000.0
+    PRATEsfc = PRATEsfc.assign_attrs(units="kg/m^2/s")
+    return PRATEsfc
+
+
+def add_additional_surface_fields(ds):
```

Comment:
Could this be re-used from the `format_output.py` script or moved to `util.py`?

Reply:
Moved to `util.py`